src/main/frontend/pipeline-console-view/pipeline-console/main/StageNodeLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export interface StageNodeLinkProps {
+    agent: string;
+}
+
+function getAgentUrl(name: string) {
+    // Wrap built-in in brackets
+    const id = name == "built-in" ? "(built-in)" : name;
+    const rootPath = document.head.dataset.rootUrl
+    return `${rootPath}/computer/${id}/`;
+}
+
+const StageNodeLink = ({agent}: StageNodeLinkProps) => {
+    const agentName = agent == "built-in" ? "Jenkins" : agent;
```

Comment:
Doesn't seem to be working the agent name is coming through as `jenkins` and not `Jenkins`:

![image](https://github.com/user-attachments/assets/04b90cfd-afa4-4128-80a4-f1008a69c2bf)